docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -0,0 +1,91 @@
+# Blockchain API Server
+
+## Blockchain RPC
+
+Blockchain RPC API for interacting with different blockchains can be found in a separate `Blockchain RPC` subsection.
```

Comment:
We have a [separate section for the Blockchain RPCs](https://github.com/WalletConnect/walletconnect-specs/tree/main/docs/blockchain-rpc)